### DISCUSSION TITLE
Incorrect documentation for join function

### DISCUSSION THREAD

**Discussion by ePirat** [community/community#25106] [Actions]
<p>The join function documented <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/contexts-and-expression-syntax-for-github-actions#join" rel="nofollow noopener">here</a> does not behave as documented.</p>
<p>Per the documentation, join([‘Hello’, ‘World’]) returns ‘Hello World’, but actually this returns ‘Hello,World’.</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Thanks <a class="mention" href="/epirat">@epirat</a>  for reaching this out!  The document for ‘join’ has a bug, it’s being tracked internally now.</p>
<p><img src="https://user-images.githubusercontent.com/52772761/181088785-0a243d3e-a983-4080-9566-fa68e89f03e7.png" alt="join.png" data-base62-sha1="vzpb2GWPswz7pPP1ZSJb1vCCzqy" width="669" height="141"></p>
